internal/icinga2/client_api.go

```diff
@@ -130,6 +130,31 @@ func (client *Client) queryObjectsApiDirect(ctx context.Context, objType, objNam
 		map[string]string{"Accept": "application/json"})
 }
 
+// fetchIcingaAppStatus retrieves the global state of the IcingaApplication type via the /v1/status endpoint.
+func (client *Client) fetchIcingaAppStatus(ctx context.Context) (*IcingaAppStatus, error) {
+	response, err := client.queryObjectsApi(
+		ctx,
+		[]string{"/v1/status/IcingaApplication/"},
```

Comment:
This will require additional permissions for the Icinga 2 API user. At the moment, `status/query` is not part of the permissions list, <https://icinga.com/docs/icinga-notifications-web/latest/doc/03-Configuration/#adding-an-icinga-2-source>.

Thus, the documentation over at :spider_web: needs an update.

Reply:
> Thus, the documentation over at 🕸️ needs an update.

That's not something to be done with this PR! But once this is merged, I will open an issue there.

Reply:
Done https://github.com/Icinga/icinga-notifications-web/issues/228